ast/base.go

```diff
@@ -95,7 +95,3 @@ type funcNode struct {
 
 // functionExpression implements FunctionNode interface.
 func (fn *funcNode) functionExpression() {}
-
-type resultSetNode struct {
-	resultFields []*ResultField
-}
```

Comment:
PTAL @XuHuaiyu pingcap/tidb#4988. Is it safe to remove this inner struct given that nothing on `ast.ResultSetNode` relies on ResultFields anyway?